theano/tensor/nnet/bn.py

```diff
@@ -413,15 +413,27 @@ def infer_shape(self, node, shape):
     def make_node(self, x, scale, bias, epsilon=1e-4,
                   running_average_factor=0.1,
                   running_mean=None, running_var=None):
+        x = as_tensor_variable(x)
+        scale = as_tensor_variable(scale)
+        bias = as_tensor_variable(bias)
+        epsilon = as_tensor_variable(epsilon)
```

Comment:
This can cause trouble with floatX=float32 and epsilon=1e-5.  I don't know if we want to address those.